pyanaconda/modules/payloads/payload/dnf/dnf.py

```diff
@@ -370,6 +372,25 @@ def calculate_required_space(self):
         required_space = calculate_required_space(self.dnf_manager)
         return required_space.get_bytes()
 
+    def collect_requirements(self):
+        """Return installation requirements for this module.
+
+        :return: a list of requirements
+        """
+        requirements = []
+
+        if self.dnf_manager.is_package_available("dnf5"):
+            plugins_name = "dnf5-modules"
+        else:
+            plugins_name = "dnf-plugins-core"
+
+        if self._packages_configuration.multilib_policy != MULTILIB_POLICY_BEST:
+            requirements.append(
+                Requirement.for_package(plugins_name, reason="Needed to enable multilib support.")
+            )
+
+        return requirements
+
```

Comment:
This might not be the correct way at the end. I looked on the code and it seems that we should follow path:
https://github.com/rhinstaller/anaconda/blob/26a6ecdc6c188849e1264b278104ab1e7ad6de56/pyanaconda/modules/payloads/payload/dnf/installation.py#L132